docs/Makefile

```diff
@@ -1,3 +1,19 @@
+########################################################################################################################
+##### CUSTOM CONFIGURATION
+VENV = env/bin/activate
+PORT = 8000
+# CORE COMMANDS
+install:
+	##### setup virtualenv
```

Comment:
could you pls replace tabs with whitespaces entirely in all file?
and pls check if your .editorconfig properly configured